packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../common/common';
+
+$defaultColor: $ms-color-neutralTertiary;
+$selectedColor: $ms-color-black;
+$hoverColor: $ms-color-themeDarkAlt;
+$disabledColor: $ms-color-neutralTertiaryAlt;
+
+// Hide input field
+.ms-Rating-input {
+  position: absolute;
+  opacity: 0;
+  top: 0px;
+}
+
+.ms-Rating-container {
```

Comment:
Is this container necessary? Seems like the stars could be direct children of `ms-Rating`.